src/LogBuffer.cpp

```
@@ -0,0 +1,68 @@
+#include "LogBuffer.h"
+#include <stdarg.h>
```

Clang (14.0.6)

[warning] inclusion of deprecated C++ header 'stdarg.h'; consider using 'cstdarg' instead

(modernize-deprecated-headers)